### ISSUE TITLE
Bug: Power Control should allow built-in volume indicator on desktop

### ISSUE THREAD

**Issue by thw0rted** [ayufan/steam-deck-tools#60]
When Power Control is running, the built-in volume indicator is not shown.  That makes sense when the current volume is visible in the OSD, or at least can be called up by holding the 3-dots button to expand the overlay.  But if no game is running, the OSD is not visible, and there is no indication of the current volume when you push the volume buttons.

Whatever mechanism is being used to suppress the built-in volume indicator, it should only be used when the OSD is visible (during gameplay).

**Comment by ayufan** [comment]
@thw0rted Yes, this is correct. Since the app wants to change volume by 5, it does intercept volume keys, and does not pass to explorer that handles this. Unsure, this is hard to fix.

**Comment by thw0rted** [comment]
Would it make sense to only hook the volume keys when launching a game (OSD start), then unhook them when the game ends (OSD stop)?  I'm not sure if you already get some kind of event when the OSD "kicks in".

**Comment by ayufan** [comment]
Possible, but this is also the whole reason it was added: the volume controls in Windows are annoyingly slow. Anyway, we will see, maybe the better is to passthrough to Windows, but quickly change the value.

**Comment by ayufan** [comment]
For others. If you are annoyed by this, configure in `PowerControl.dll.ini` the `EnableVolumeControls=False`.
